README-REST.md

```diff
@@ -0,0 +1,1122 @@
+
+<table cellspacing="0" cellpadding="0" border="0" >
+	<tr>
+		<td>
+			<table cellspacing="3" border="0">
+				<tr>
+					<td><a href="https://bitvavo.com"><img src="./assets/bitvavo-mark-square-blue.svg" width="100" title="Bitvavo Logo"/></td>
+					<td><h1>Bitvavo SDK for Python</h1></td>
+				</tr>
+			</table>
+		</td>
+	</tr>
+</table>
+
+Crypto starts with Bitvavo. You use Bitvavo SDK for Python to buy, sell, and store over 200 digital assets on Bitvavo from inside your own app. 
+
+To trade and execute your advanced trading strategies, Bitvavo SDK for Python is a wrapper that enables you to easily call every endpoint in [Bitvavo API](https://docs.bitvavo.com/).
+
+- [Prerequisites](#prerequisites) - what you need to start developing with Bitvavo SDK for Python. 
+- [API reference](#api-reference) - in-depth information about Bitvavo SDK for Python
+
+This page gives reference information for REST calls made using Bitvavo SDK for Python.
+
+## Prerequisites
+
+To start programming with Bitvavo SDK for Python you need:
+
+- [Python3](https://www.python.org/downloads/) installed on your development environment
+
+   If you are working on MacOS, ensure that that you have installed SSH certificates:
+   ```terminal
+    open /Applications/Python\ 3.12/Install\ Certificates.command
+    open /Applications/Python\ 3.12/Update\ Shell\ Profile.command
+    ```
+- A Python app. Use your favorite IDE, or run from the command line
+- An [API key and secret](https://support.bitvavo.com/hc/en-us/articles/4405059841809) associated with your Bitvavo account
+
+  You control the actions your app can do using the rights you assign to the API key. Possible rights are:
+  - **View**: retrieve information about your balance, account, deposit and withdrawals.
+  - **Trade**: place, update, view and cancel orders.
+  - **Withdraw**: withdraw funds.
+
+       Best practice is to not grant his privilege, withdrawals using the API do not require 2FA and e-mail confirmation.
+
+## API reference
+
+This is the python wrapper for the Bitvavo API. This project can be used to build your own projects which interact with the Bitvavo platform. Every function available on the API can be called through a REST request or over websockets. For info on the specifics of every parameter consult the [Bitvavo API documentation](https://docs.bitvavo.com/)
```

Comment:
In terms of spelling, I would appreciate if we case as follows:
- `Python` with a capital P as it is a name
- `WebSocket` with capital W and S as it is spelled officially, see also [Wikipedia](https://en.wikipedia.org/wiki/WebSocket)
```suggestion
This is the Python wrapper for the Bitvavo API.
This project can be used to build your own projects which interact with the Bitvavo platform.
Every function available on the API can be called through a REST request or over WebSockets.
For info on the specifics of every parameter consult the [Bitvavo API documentation](https://docs.bitvavo.com/)
```

Reply:
Done.